.travis.yml

```diff
@@ -13,22 +13,26 @@ php:
     - 7.0
     - hhvm
 
+env:
+    - COMPOSER_COMMAND=install
```

Comment:
given that the lock file is not commited, this is useless. ``composer install`` without a lock file is actually running an update